.github/CODEOWNERS

```diff
@@ -105,7 +105,7 @@
 /avm/res/network/public-ip-address/ @Azure/avm-res-network-publicipaddress-module-owners-bicep @Azure/avm-core-team-technical-bicep
 /avm/res/network/public-ip-prefix/ @Azure/avm-res-network-publicipprefix-module-owners-bicep @Azure/avm-core-team-technical-bicep
 /avm/res/network/route-table/ @Azure/avm-res-network-routetable-module-owners-bicep @Azure/avm-core-team-technical-bicep
-#/avm/res/network/service-endpoint-policy/ @Azure/avm-res-network-serviceendpointpolicy-module-owners-bicep @Azure/avm-core-team-technical-bicep
+/avm/res/network/service-endpoint-policy/ @Azure/avm-res-network-serviceendpointpolicy-module-owners-bicep @Azure/avm-core-team-technical-bicep
```

Comment:
Ah wait, overlooked this. The team either is not existing, was not added to the hierachy of teams yet, or wasn't confirmed in set hierachy.

Reply:
The requirements are described [here](https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-snfr20---category-contributionsupport---github-teams-only), yet it comes down to creating both an owner & contributor team as per the shown naming convention and adding them to the correct 'parent' team so that the permissions in the BRM repository are configured correctly.

Reply:
Check now

Reply:
@krbar could you provide some guidance on this one? Other than me, you have contributed a gazillion of modules and know the process inside out :D

Reply:
I think your teams are missing a parent team. The correct parents are:
- parent of the owners team: `avm-technical-reviewers-bicep`
- parent of the contributors team: `avm-module-contributors-bicep`

You can set the parent team via team settings. Please keep in mind that the parentship must be approved by someone from the parent team and once approved, you will see an indication of the team path (see below example for the bastion host team):

<img width="508" alt="image" src="https://github.com/Azure/bicep-registry-modules/assets/20225789/36f71a52-a464-49a9-a244-72bbe0196a8b">

See also the specs for more detailed info: https://azure.github.io/Azure-Verified-Modules/specs/shared/#team-memberships

Reply:
I have done all of this. Now, waiting for the approval from the parent team.